### PR TITLE
[HH-4][FOLLOW] Create follow module

### PR DESCRIPTION
**Pull request by cristighr1995**

### CODE REVIEW

File: holiholic_follow/src/main/java/com/holiholic/follow/constant/Constants.java [hunk 1]
```diff
@@ -4,8 +4,8 @@
  *
  */
 public class Constants {
-    public final static String GET_TOPICS_URL = "http://localhost:8093/getTopics?md5Key=";
-    public final static String UPDATE_TOPICS_URL = "http://localhost:8093/updateTopics";
-    public final static String GET_PEOPLE_URL = "http://localhost:8093/getPeople?md5Key=";
-    public final static String UPDATE_PEOPLE_URL = "http://localhost:8093/updatePeople";
+    public final static String GET_TOPICS_URL = "http://localhost:8092/getTopics?md5Key=";
```
Comment (roxanamacarie) [inline holiholic_follow/src/main/java/com/holiholic/follow/constant/Constants.java:7]: Why 8092, isn't the login module running at this port?

Comment (cristighr1995) [reply holiholic_follow/src/main/java/com/holiholic/follow/constant/Constants.java:7]: follow module runs on port `8093` but this url is used to connect to the database which runs on port `8092`

File: holiholic_follow/src/main/java/com/holiholic/follow/follow/PeopleManager.java [hunk 2]
```diff
@@ -27,18 +27,21 @@ public static void setLogger() {
...
+    public static boolean updatePeople(JSONObject request, boolean follow) {
         try {
-            String md5Key = jsonObject.getString("md5Key");
-            String followersMd5Key = jsonObject.getString("followersMd5Key");
+            String md5KeyFrom = request.getString("md5KeyFrom");
+            String md5KeyTo = request.getString("md5KeyTo");
+            String operation = follow ? "follow" : "unfollow";
 
-            LOGGER.log(Level.FINE, "New request from [0] to follow [1]", new Object[]{md5Key, followersMd5Key});
+            LOGGER.log(Level.FINE, "New request from {0} to {1} {2}", new Object[]{md5KeyFrom, md5KeyTo});
```
Comment (roxanamacarie) [inline holiholic_follow/src/main/java/com/holiholic/follow/follow/PeopleManager.java]: It should be: ```LOGGER.log(Level.FINE, "New request from {0} to {1} {2}", new Object[]{md5KeyFrom, operation, md5KeyTo});```

Comment (cristighr1995) [reply holiholic_follow/src/main/java/com/holiholic/follow/follow/PeopleManager.java]: yes

File: holiholic_follow/src/main/java/com/holiholic/follow/constant/Constants.java (hunk 1 above)
Comment (roxanamacarie) [reply holiholic_follow/src/main/java/com/holiholic/follow/constant/Constants.java:7]: Ah, ok 👍

**Review by roxanamacarie — APPROVED** [review]
👍

**Pull request merged**